ui/src/pages/repos/index.tsx

```diff
@@ -270,7 +285,27 @@ export default function Page() {
         👋 Welcome, {me?.firstname}! Please open or create a repository to get
         started.
       </Box>
-      <Repos />
+      {error && <Alert severity="error">{error}</Alert>}
+      {!error && loading && (
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "center",
+          }}
+        >
+          <CircularProgress />
+        </Box>
+      )}
+      <Repos
+        callback={(arg) => {
+          if (arg) {
+            setError(arg);
+          }
+        }}
+        onLoading={(value) => {
+          setLoading(value);
+        }}
+      />
```

Comment:
What was the issue here?

Reply:
there is no issue, i just find that there are two loading hint. so i fixed it in the same time

Reply:
I see, let’s remove this, because this adds extra complexity. We’ll redirect to login page anyway, so there won’t be warnings. #77

Reply:
No worry. I'm taking over this part. But I want to confirm if it redirects the login page immediately or gives any hints and then jump to the login page?

Reply:
normally, it needs show hint firstly and then redirects